index.js

```diff
@@ -29,12 +29,12 @@ module.exports = function (app, config) {
     }
     if (typeof serviceCfg === 'string') {
       option.endpoint =  serviceCfg;
-      option.accessKeyId = 'anonymous';
-      option.accessKeySecret = app.config.systemToken;
+      option.accessKeyId = app.config.defaultAccessKeyId || 'anonymous';
+      option.accessKeySecret =  app.config.systemToken || app.config.defaultAccessKeySecret;
```

Comment:
judge defaultAccessKeySecret first
```suggestion
       option.accessKeySecret =  app.config.defaultAccessKeySecret || app.config.systemToken;
```